test/unit/capsize_deployer_test.rb

```diff
@@ -261,6 +261,24 @@ def test_stage_file_is_written
     remove_directory(@deployment.stage.project.capsize_project_name)
   end
 
+  def test_configuration_parameters_written_correctly
+    deployment = create_new_deployment(:stage => @stage)
+
+    @stage.project.configuration_parameters.create(:name => 'app_name', :value => 'madeupname')
+    @stage.project.configuration_parameters.create(:name => 'keep_releases', :value => '8')
+    @stage.project.configuration_parameters.create(:name => 'linked_dirs', :value => '%w{ made_up_dir }')
+
+    write_deployer_files(deployment)
+    file = File.open(rooted("#{@stage.project.capsize_project_name}/deploy.rb"))
+    contents = file.read
+
+    assert_match /set :app_name, 'madeupname'/, contents
+    assert_match /set :keep_releases, 8/, contents
+    assert_match /set :linked_dirs, %w{ made_up_dir }/, contents
```

Comment:
You probably want to ensure these are on their own line (using `^` and `$` and the `m` regex modifier, possibly with some `\s*` tossed in for good measure). Also, add some trickier cases, such as `"23skidoo"`, `"foo2"`, `nil`, etc. For bonus points, make a single method which takes a name, a value, and an expected string, and then a series of tests which call into that.